### PR TITLE
[IMP] maintainer-quality-tools: Enable installing 0.12.1 version of webkit qt patched

### LINKED ISSUE #227: Enable installing 0.12.1 version of webkit qt patched
Currently we are using the github generic url to install wkhtmltopdf patched but the odoo official version is 0.12.1 and we need supported it.

### PR DESCRIPTION
**Pull request by mapuerta**
This PR resolves fix #227

### CODE REVIEW

**Comment by mapuerta** [comment]
@moylop260 Ready for review

1. WKHTMLTOPDF_VERSION="0.12.1"



result_version_0_12_1

shoed_version_0_12_1


2. WKHTMLTOPDF_VERSION="0.12.3"



show_version_0_12_3

**Comment by mapuerta** [comment]
@moylop260 Ready for review

without new methods
Here [video](https://youtu.be/7FJPdY2BBDE) demonstrative

File: travis/travis_install_nightly [hunk 1]
```diff
@@ -211,7 +211,22 @@ if [ "x${WKHTMLTOPDF_VERSION}" == "x" ] ; then
     WKHTMLTOPDF_VERSION="0.12.3"
 fi
 echo "Installing the version of webkit ${WKHTMLTOPDF_VERSION}"
-(cd ${HOME}/maintainer-quality-tools/travis/ && wget -qO- -t 1 --timeout=240 https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/${WKHTMLTOPDF_VERSION}/wkhtmltox-${WKHTMLTOPDF_VERSION}_linux-generic-amd64.tar.xz | tar -xJ --strip-components=2 wkhtmltox/bin/wkhtmltopdf)
+
+if [[ ${WKHTMLTOPDF_VERSION} == "0.12.3" ]]
```
Comment (moylop260) [inline travis/travis_install_nightly]: And for 0.12.4?

Comment (mapuerta) [inline travis/travis_install_nightly]: @moylop260 This issue https://github.com/Vauxoo/maintainer-quality-tools/issues/227 was open for add support the version 0.12.1. Now also is need add support the version 0.12.4 ?

**Comment by mapuerta** [comment]
@moylop260 Ready for review 

[video](https://www.youtube.com/watch?v=USYO5BvPi14&feature=youtu.be)

**Comment by moylop260** [comment]
The projects are using `apt-get install wkhtmltopdf` from `.travis.yml`
Could you check if this script is compatible with this way?
I mean,
Step 1:
 - `apt-get install wkhtmltopdf`
Step 2:
 - `WKHTMLTOPDF_VERSION=0.12.1 travis_install_nightly`

Because I don't remember what is the binary path that is used from apt-get and we could have 2 /usr/bin or /usr/local/bin paths (Or you can use MQT folder that is used as the first one)

**Comment by mapuerta** [comment]
@moylop260 You mean this [video](https://www.youtube.com/watch?v=TyrcK1dG03I)

**Comment by moylop260** [comment]
Yeah, that is the expected behaviour thank you!

**Comment by moylop260** [comment]
@mapuerta Could you migrate this PR to OCA/mqt project, please?

**Pull request merged**